### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by andrewkern**
adding a default code of conduct to the project. we should probably edit this further, but wanted to start a PR and get people's takes

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Andrew Kern or Jerome Kelleher.
+
+
+
+Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
+
+## 8. Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify PopSim Consortium with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies.
```
Comment (dschride) [inline CODE_OF_CONDUCT.md]: This code of conduct looks mostly fine to me except this statement: "Your grievance will be handled in accordance with our existing governing policies." As far as I know we don't have any such existing governing policies.

Comment (andrewkern) [reply CODE_OF_CONDUCT.md]: we might need some...

**Comment by jeromekelleher** [comment]
Seems good to me, if possibly a little beyond our current needs. We probably need to talk about governance etc; something to talk about at today's meeting.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (dschride) [reply CODE_OF_CONDUCT.md]: Maybe just change to "Your grievance will be considered by the organizers." or something similar.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Citizen Code of Conduct
+
+## 1. Purpose
+
+A primary goal of PopSim Consortium is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
```
Comment (dschride) [inline CODE_OF_CONDUCT.md]: add something like "If you choose to participate in the group then you are implicitly agreeing to abide by this code of conduct."

Comment (andrewkern) [reply CODE_OF_CONDUCT.md]: done

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (andrewkern) [reply CODE_OF_CONDUCT.md]: done

**Comment by dschride** [comment]
All good now as far as I'm concerned.

**Comment by jeromekelleher** [comment]
OK, merging. We can tweak later, but this is a great foundation. Thanks for getting this set up @andrewkern.

**Pull request merged**